megamek/src/megamek/client/Client.java

```diff
@@ -1186,10 +1182,14 @@ private void cacheImgTag(Entity entity){
      * Gets the current mech image
      */
     private Image getTargetImage(Entity e){
-        if(e.isDestroyed()) {
-            return bv.getTilesetManager().wreckMarkerFor(e, -1);
+        if(bv == null){
+            return  null;
         }else {
-            return bv.getTilesetManager().imageFor(e);
+            if (e.isDestroyed()) {
+                return bv.getTilesetManager().wreckMarkerFor(e, -1);
+            } else {
+                return bv.getTilesetManager().imageFor(e);
+            }
         }
```

Comment:
```
if (bv == null) {
    return null;
} else if (e.isDestroyed()) {
    return bv.getTilesetManager().wreckMarkerFor(e, -1);
} else {
    return bv.getTilesetManager().imageFor(e);
}
```

Reply:
Suggested rewrite to improve readability.

Reply:
Apparently I'm sloppy when it's early. I appreciate the suggestions.

Reply:
It happens to us all, and welcome :)